nucypher/cli/worklock.py

```diff
@@ -0,0 +1,210 @@
+@group_worklock_options
+@group_general_config
+def claim(general_config, worklock_options, registry_options, force):
+    emitter = _setup_emitter(general_config)
+    if not worklock_options.bidder_address:
+        worklock_options.bidder_address = select_client_account(emitter=emitter,
+                                                                provider_uri=general_config.provider_uri)
+    if not force:
+        emitter.message("Note: Claiming WorkLock NU tokens will initialize a new stake.", color='blue')
+        click.confirm(f"Continue worklock claim for bidder {worklock_options.bidder_address}?", abort=True)
+    emitter.message("Submitting Claim...")
+    registry = registry_options.get_registry(emitter, general_config.debug)
+    worklock_agent = worklock_options.create_agent(registry=registry)
+    receipt = worklock_agent.claim(bidder_address=worklock_options.bidder_address)
+    paint_receipt_summary(receipt=receipt, emitter=emitter, chain_name=worklock_agent.blockchain.client.chain_name)
+    paint_worklock_claim(emitter, bidder_address=worklock_options.bidder_address)
+    return  # Exit
+
+
+@worklock.command()
+@group_worklock_options
+@group_registry_options
+@group_general_config
+def remaining_work(general_config, worklock_options, registry_options):
+    emitter = _setup_emitter(general_config)
+    if not worklock_options.bidder_address:
+        worklock_options.bidder_address = select_client_account(emitter=emitter,
+                                                                provider_uri=general_config.provider_uri)
+    registry = registry_options.get_registry(emitter, general_config.debug)
+    worklock_agent = worklock_options.create_agent(registry=registry)
+    _remaining_work = worklock_agent.get_remaining_work(bidder_address=worklock_options.bidder_address)
+    emitter.message(f"Work Remaining for {worklock_options.bidder_address}: {_remaining_work}")
+    return  # Exit
+
+
+@worklock.command()
+@option_force
+@group_registry_options
+@group_worklock_options
+@group_general_config
+def refund(general_config, worklock_options, registry_options, force):
+    emitter = _setup_emitter(general_config)
+    if not worklock_options.bidder_address:
+        worklock_options.bidder_address = select_client_account(emitter=emitter,
+                                                                provider_uri=general_config.provider_uri)
+    if not force:
+        click.confirm(f"Collect ETH refund for bidder {worklock_options.bidder_address}?", abort=True)
+    emitter.message("Submitting WorkLock refund request...")
+    registry = registry_options.get_registry(emitter, general_config.debug)
+    worklock_agent = worklock_options.create_agent(registry=registry)
+    receipt = worklock_agent.refund(bidder_address=worklock_options.bidder_address)
+    paint_receipt_summary(receipt=receipt, emitter=emitter, chain_name=worklock_agent.blockchain.client.chain_name)
+    return  # Exit
+
+
+@worklock.command()
```

Comment:
Because of the underscore vs dash, the name needs to be defined.

```suggestion
@worklock.command(name='burn-unclaimed-tokens')
```

Reply:
ignore